webview/src/plots/components/styles.module.scss

```diff
@@ -12,6 +23,18 @@ $gap: 20px;
   width: 100%;
 }
 
+.plotsSectionMessage {
```

Comment:
Not sure if the message looks better or worse without borders:

<img width="1029" alt="Screenshot 2023-03-28 at 9 50 38 AM" src="https://user-images.githubusercontent.com/43496356/228311298-4d69049b-f4f5-4c36-80f2-2b4a718c9398.png">